base/applications/rapps/gui.cpp

```diff
@@ -59,6 +65,14 @@ enum SNPSHT_STATUS
 
 #define PI 3.1415927
 
+typedef struct __SnapshotDownloadParam
+{
+    LONGLONG ID;
+    HANDLE hFile;
+    HWND hwndNotify;
+    ATL::CStringW DownloadFileName;
+}SNPSHT_DOWNLOAD_PARAM;
```

Comment:
```suggestion
} SNPSHT_DOWNLOAD_PARAM;
```

Reply:
Also, a name like `SnapshotDownloadParam` is perfectly fine.
The all-caps and abbreviated names are a Microsoft thing, we don't have to do that for stuff like our own RApps :)